src/components/Envelope.vue

```diff
@@ -44,18 +44,24 @@
 					@click.shift.prevent="onSelectMultiple" />
 			</p>
 		</div>
-		<div class="app-content-list-item-line-one" :title="addresses">
+		<div class="app-content-list-item-line-one"
+			:title="addresses"
+			role="heading"
+			:aria-label="3">
```

Comment:
```suggestion
			:aria-level="3">
```

Just a slight error – this needs to be level instead of label – label is the actual description. :)

Reply:
Ah and these should also go into the router-link instead, to only have it once instead of 3 times.